MIGRATION.md

```diff
@@ -130,11 +130,13 @@ module.exports = {
       rules: [
         ...config.module.rules,
         {
+          test: /\.stories\.tsx?$/,
```

Comment:
If the test is not provided, the build process will take way longer than it's supposed to